po/de.po

```diff
@@ -0,0 +1,2393 @@
+# German translations for Ledger package.
```

Comment:
Several words like `commodity` have not (yet) been translated. This was already quite a bit of work as it is and my brain went 👻 while thinking in English when working on this PR and switching to translation mode when expanding on this file.